data/blog/8th-wave-of-bitcoin-grants.mdx

```diff
@@ -0,0 +1,141 @@
+implementations, promoting greater interoperability, customization, and
+experimentation within the Bitcoin and lightning communities.
+
+With the support of this grant, Lampo will focus on developing key features,
+including a Proof-of-Concept (PoC) for the VLS signer, integration of LDK with
+LNprototest, and the eventual implementation of BOLT12. The project will
+prioritize bug fixes, improve Lampo's documentation, and introduce new tutorials
+to foster greater accessibility and collaboration. By creating a truly
+open-source lightning node, Lampo supports the decentralization of financial
+infrastructure and drives innovation in lightning technology.
+
+Repository: [vincenzopalazzo/lampo.rs](https://github.com/vincenzopalazzo/lampo.rs)  
+License: BSD 3-Clause
+
+### Cashu Nutshell
+
+Cashu Nutshell is an open-source Chaumian ecash system focused on enhancing
+privacy for users of custodial Bitcoin applications. The system enables users to
+make anonymous transactions without revealing their identity, balance, or
+transaction history. Leveraging Cashu mints allows for private ecash exchanges,
+preventing censorship and blocking of specific users. The project also
+integrates with decentralized platforms like nostr, introducing features such as
+private ecash receipts and [Proof of Liabilities
+schemes](https://gist.github.com/callebtc/ed5228d1d8cbaade0104db5d1cf63939),
+which allow users to confirm transactions and monitor mint operations without
+compromising privacy.
+
+With the support of this grant, Cashu Nutshell will implement [synthetic USD
+e-cash](https://github.com/cashubtc/nutshell/pull/608) backed by LNMarkets,
+providing a privacy-focused alternative to Tether for users in regions with
+unstable fiat currencies. Key milestones include the development of [Discreet
+Log Contracts](https://github.com/cashubtc/nutshell/pull/576) for ecash, and the
+introduction of [private ecash
+receipts](https://github.com/cashubtc/nuts/pull/155) using nostr.
+
+Repository: [cashubtc/nutshell](https://github.com/cashubtc/nutshell)  
+License: MIT
+
+### PickhardtPayments Plugin
+
+PickhardtPayments, originally developed by [Rene
+Pickhardt](https://opensats.org/blog/rene-pickhardt-receives-lts-grant) and
+Stefan Richter, is a payment routing strategy aimed at improving the reliability
+and efficiency of payments on the Lightning Network (LN). This plugin project
+focuses on implementing the Pickhardt-Richter model into Core-Lightning via the
+"renepay" plugin. The model uses advanced Bayesian analysis and Minimum Cost
+Flow ([MCF](https://github.com/Lagrang3/mcf-algorithms)) algorithms to calculate
+optimal Multi-Path Payments (MPP), reducing payment failures due to liquidity
+shortages, particularly for smaller and less connected nodes in the network.
+
+With the support of this grant, the project will further develop and split the
+payment process into two plugins—"askrene" for route computation and "renepay"
```

Comment:
Here and on the next line, use the em dash (—) instead, a long dash that is used in a sentence to separate extra information, expand on a clause, or indicate a break.

Reply:
Ah thanks. Thought my dash was long enough to be Em.